djitellopy/tello.py

```diff
@@ -1057,14 +1051,28 @@ def start(self):
         self.worker.start()
 
     def update_frame(self):
-        """Thread worker function to retrieve frames from a VideoCapture
+        """Thread worker function to retrieve frames using PyAV
         Internal method, you normally wouldn't call this yourself.
         """
         while not self.stopped:
-            if not self.grabbed or not self.cap.isOpened():
-                self.stop()
-            else:
-                self.grabbed, self.frame = self.cap.read()
+            
+            try:
+                for frame in self.container.decode(video=0):
+                    self.frame = cv2.cvtColor(np.array(frame.to_image()), cv2.COLOR_RGB2BGR)
```

Comment:
Is this the only place where we still use opencv after your merge? I assume this makes sure the image is still BGR like it was before when we used opencv for grabbing frames?
I would be OK with removing this line, changing the pixel order even though it means breaking existing programs that depend on the pixel order. The OpenCV lib in `requirenments.txt` is pretty outdated and would require updating with potential braking changes anyways.